ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md

```diff
@@ -3961,6 +3961,51 @@ interface Ethernet81/2
 interface Ethernet81/10
    description isis_port_channel_member
    channel-group 110 mode active
+!
+interface Ethernet82
+   description Switchport_tap_tool
+   switchport tap native vlan 10
+   switchport tap identity 3 inner 5
+   switchport tap mac-address dest 01:00:00:00:00:00 src 01:23:45:67:89:ab
+   switchport tap mpls pop all
```

Comment:
I have a different order on my device 4.33

> `switchport tap mpls pop all` is after all the encap

```
interface Ethernet2
   description Switchport_tap_tool
   switchport tap native vlan 10
   switchport tap identity 3 inner 5
   switchport tap mac-address dest 01:00:00:00:00:00 src 01:23:45:67:89:ab
   switchport tap encapsulation gre protocol 0x1 strip
   switchport tap encapsulation gre protocol 0x2 feature header length 3 strip
   switchport tap encapsulation gre protocol 0x3 feature header length 2 strip re-encapsulation ethernet
   switchport tap encapsulation gre protocol 0x4 strip re-encapsulation ethernet
   switchport tap encapsulation gre destination 1.1.1.1 source 1.1.1.2 protocol 0x0 strip
   switchport tap encapsulation gre destination 1.1.1.1 source 1.1.1.2 strip
   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x10 strip
   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x11 feature header length 2 strip re-encapsulation ethernet
   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x12 strip re-encapsulation ethernet
   switchport tap encapsulation gre destination 2.1.1.3 source 2.1.1.4 strip
   switchport tap mpls pop all
   switchport tool mpls pop all
   switchport tool encapsulation vn-tag 
```

Reply:
Yes the order for "mpls_pop_all"  is not correct here 

I have mentioned it in the template -
```
{# The position of "mpls_pop_all" does not match the EOS CLI to implement this "elif" with "vxlan_strip" #}
{%         if ethernet_interface.switchport.tap.mpls_pop_all is arista.avd.defined(true) %}
   switchport tap mpls pop all
{%         elif ethernet_interface.switchport.tap.encapsulation.vxlan_strip is arista.avd.defined(true) %}
   switchport tap encapsulation vxlan strip
{%         endif %}
```